src/CommunityToolkit.Aspire.Hosting.Ollama/PublicAPI.Shipped.txt

```diff
@@ -12,7 +12,6 @@ Aspire.Hosting.ApplicationModel.OpenWebUIResource.OpenWebUIResource(string! name
 Aspire.Hosting.ApplicationModel.OpenWebUIResource.PrimaryEndpoint.get -> Aspire.Hosting.ApplicationModel.EndpointReference!
 Aspire.Hosting.OllamaResourceBuilderExtensions
 static Aspire.Hosting.OllamaResourceBuilderExtensions.AddModel(this Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.OllamaResource!>! builder, string! modelName) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.OllamaResource!>!
-static Aspire.Hosting.OllamaResourceBuilderExtensions.AddOllama(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name = "Ollama", int? port = null, string! modelName = "llama3") -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.OllamaResource!>!
```

Comment:
Please revert this.

Reply:
ah, must have missed a commit sha to cherry-pick

Reply:
Fixed